server/world/chunk/light_area.go

```diff
@@ -132,13 +132,19 @@ func (a *lightArea) iterEdges(filter func(a, b *SubChunk) bool, f func(a, b cube
 	}
 }
 
-// iterHeightmap iterates over the heightmap of the lightArea and calls the function f with the heightmap value, the
-// heightmap value of the highest neighbour and the Y value of the highest non-empty SubChunk.
+// iterHeightmap iterates over the HeightMap of the lightArea and calls the function f with the HeightMap value, the
+// HeightMap value of the highest neighbour and the Y value of the highest non-empty SubChunk.
```

Comment:
Would revert these heightmap -> HeightMap changes as it doesn't refer to the HeightMap method.